docs/source/federated-types/sharing-types.mdx

```diff
@@ -0,0 +1,404 @@
+---
+title: Sharing types and fields
+sidebar_title: Sharing types and fields
+description: Sharing types and fields across multiple subgraphs
+---
+
+import { CodeColumns } from 'gatsby-theme-apollo-docs';
+
+> ⚠️ The field sharing model has changed significantly in Federation 2. For a summary of these changes, [see what's new](/federation-2/new-in-federation-2/#type-and-field-sharing). 
+>
+>For more information about using value types in Federation 1, you can view the [previous version of this article here](/federation/value-types/).
+
+In a federated graph, it's common to want to reuse a GraphQL type in multiple [subgraphs](./subgraphs/). 
+
+For example, suppose you want to define and reuse a generic `Position` type in different subgraphs:
+
+```graphql
+type Position {
+  x: Int!
+  y: Int!
+}
+```
+
+This article describes how to share a [GraphQL Object type's](https://www.apollographql.com/docs/apollo-server/schema/schema/#object-types) fields in a federated graph, allowing multiple subgraphs to **redefine** and **resolve** those shared fields. 
+
+## Using `@shareable`
+
+The `@shareable` directive enables multiple subgraphs to resolve a particular field, or set of fields, on an Object type across multiple subgraphs.
+
+The `@shareable` directive is new in Federation 2. To enable it, you first need to add the following snippet to the top of **all of your subgraph files** to [opt-in to Federation 2](../federation-2/moving-to-federation-2/#opt-in-to-federation-2):
```

Comment:
I was wrong about "all subgraph files" being mandatory! 😬 Here's a truer sentence:

"To use it in a subgraph schema, you first need to add the following snippet to that schema to [opt in to Federation 2]: